base/operators.jl

```diff
@@ -328,9 +328,31 @@ eltype(::Type{Any}) = Any
 eltype(t::DataType) = eltype(supertype(t))
 eltype(x) = eltype(typeof(x))
 
-# function pipelining
+# function pipelining, composition & negation
+
 |>(x, f) = f(x)
 
+"""
+    ∘(f, g)
+    f ∘ g
```

Comment:
@andyferris did you include the entire signature, i.e.

```rst
.. function:: ∘(f, g)
              f ∘ g

```

or just the first line `.. function:: ∘(f, g)`? Signature matching requires the entire first code block, not just the first line of it.